ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+## Expected  behavior
+
+
+## Actual behavior
+
+
+## Steps to reproduce
```

Comment:
I would move the repro steps to be beginning, I think it makes a more natural sense and tells the story better.